### DISCUSSION TITLE
Model-based RL Algorithms

### DISCUSSION THREAD

**Discussion by lmc19970711** [opendilab/DI-engine#165] [Q&A]
Can you support more model-based algorithms, such as Dreamer, World model, etc.?

**Comment by PaParaZz1** [comment]
Yes, we are working on enriching our model-based RL algorithm zoo, we also opened source a new [repo](https://github.com/opendilab/awesome-model-based-RL) about a curated list of resources of mbrl several days ago.

We are preparing steve and mcts-based method such as MuZero, and we also welcome more collaborators, can you offer more detailed thoughts, such as:
- paper link and related code
- evaluation environment and target performance (e.g.: reward and running time)
- how can you participate in this work
- other funny tricks and detailed comments
